andela/zinnia-ah-backend
# #164797163 User can report an article

#### What does this PR do?
User can report an Article

#### Description of Task to be completed?
- [x] get an article
- [x] check if the article exists
- [x] get the report type and content.
- [x] report the article

#### How should this be manually tested?
- Clone the repo,
- Run ```npm install``` to install all dependencies,
- Create .env file using the .env.example as a guide and enter you environment variables
- Run `sequelize db:create`  
- Run `sequelize db:migrate` 
- Run `sequelize db:seed:all`   
- Run npm run start:dev,
- On Postman, send a POST request `localhost:3000/api/v1/articles/:articleId/report` supply an a report type and content in the request body
- You should get the screenshot below

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories?(if applicable)
#164797163

#### Screenshots
POSTMAN RESPONSE
<img width="1440" alt="Screenshot 2019-04-17 at 5 28 44 PM" src="https://user-images.githubusercontent.com/31692932/56304584-56914500-6136-11e9-81d6-ee0be161aec5.png">

FLOWCHART
<img width="401" alt="Screenshot 2019-04-17 at 5 30 48 PM" src="https://user-images.githubusercontent.com/31692932/56304707-9f48fe00-6136-11e9-9b90-6a13f995d696.png">
